changelog/unreleased/9155

```diff
@@ -0,0 +1,7 @@
+Bugfix: --version showed incorrect information about VFS support
+
+--version used to always show "Off", even when a VFS plugin was available.
+This has been fixed now.
+
+https://github.com/owncloud/client/pull/9155
```

Comment:
Isn't one supposed to be an issue?

Reply:
It is an issue, but I typed the URL by hand... I'll fix it.